torch/_inductor/compile_fx.py

```diff
@@ -353,9 +367,25 @@ def compile_fx_inner(
         # doesnt work for non-trees because the warmup run would apply mutation twice
         if config.triton.cudagraph_trees:
             # checking if mutation is only on parameters/static inputs
-            has_mutation = not all(
-                idx < num_fixed for idx in compiled_graph.mutated_input_idxs
-            )
+            mutation_indices = [
+                idx for idx in compiled_graph.mutated_input_idxs if idx >= num_fixed
+            ]
+            has_mutation = len(mutation_indices) != 0
+            if has_mutation:
+                stack_trace: Optional[str] = ""
+                placeholders = [
+                    node for node in gm.graph.nodes if node.op == "placeholder"
+                ]
+
+                for idx in mutation_indices:
+                    placeholder = placeholders[idx]
+                    if stack_trace := get_mutating_use_stack_trace(placeholder):
+                        break
```

Comment:
If there are multiple input mutations, this will just log the first stack trace, right? (I guess multiple input mutations on different inputs in a graph are pretty rare, just calling it out)